### ISSUE TITLE
Gengo does not seem to support when a message has a message from its own class

### ISSUE THREAD

**Issue by cjds** [fetchrobotics/rosgo#5]
Attempting to do a gengo on `GoalStatusArray`

http://docs.ros.org/melodic/api/actionlib_msgs/html/msg/GoalStatusArray.html

The result gives this error on a `go build`

```
package github.com/fetchrobotics/rosgo/test/test_action_server
	imports actionlib_tutorials
	imports actionlib_msgs
	imports actionlib_msgs
```

The reason for this is because `GoalStatusArray` has a `GoalStatus` in it. This inner message must be referenced like `GoalStatus` instead  of `actionlib_msgs.GoalStatus` to avoid the cyclic dependency

**Comment by cjds** [comment]
@pavansoundara I believe we can close this out

**Comment by pavansoundara** [comment]
Yes, this has been fixed in pr #7 .
